reana_commons/utils.py

```diff
@@ -292,8 +292,7 @@ def get_disk_usage(
     else:
         command.append("-a")
     if "Darwin" not in platform.system():
-        # Default block size in GNU is KB
-        command.append("-b")
+        command.append("--block-size=1")
```

Comment:
BTW this touches the non-Darwin code branch only. How is the disk size reporting looking on macOS? All good? No changes needed there?

Reply:
There is no `-b` or `--block-size` option on Darwin. Honestly, not sure why this condition exists in the first place as our components run in containers (hence Linux), and there is no point in calculating disk usage in `reana-client` on the user machine.

Reply:
Commit related to macOS support: https://github.com/reanahub/reana-commons/commit/ca4f3084bdb1a7e2a9e7ba874780ef81b065f46d#diff-07587e2f0e6b50d682c38a3c50b544fa214363c9631ae1d81c1923d1cfaaf44a

Not much info so it is hard to say why it was added.